sql/comments/public.sql

```diff
@@ -12,40 +12,76 @@ COMMENT ON COLUMN public.circulation_loans.item_status IS 'Last item status used
+COMMENT ON COLUMN public.circulation_loans.status_name IS 'Name of the status (currently can be any value, values commonly used are Open and Closed)';
 COMMENT ON COLUMN public.circulation_loans.system_return_date IS 'Date time when the returned item is actually processed';
 COMMENT ON COLUMN public.circulation_loans.user_id IS 'ID (generated UUID) of the patron the item was lent to. Required for open loans, not required for closed loans (for anonymization).';
 COMMENT ON COLUMN public.circulation_loans."data" IS 'JSON representation of the record';
 COMMENT ON COLUMN public.circulation_loans.claimed_returned_date IS 'Date and time the item was claimed returned for this loan';
 
 --public.circulation_requests
 COMMENT ON COLUMN public.circulation_requests.id IS 'ID (generated UUID) of the request';
+COMMENT ON COLUMN public.circulation_requests.awaiting_pickup_request_closed_date IS 'The date when the request with awaiting pickup status was closed';
 COMMENT ON COLUMN public.circulation_requests.cancellation_additional_information IS 'Potential relevant information regarding a cancellation';
 COMMENT ON COLUMN public.circulation_requests.cancellation_reason_id IS 'ID (generated UUID) of the request cancellation reason';
 COMMENT ON COLUMN public.circulation_requests.cancelled_by_user_id IS 'ID (generated UUID) of the user that cancelled the request';
 COMMENT ON COLUMN public.circulation_requests.cancelled_date IS 'Date the request was cancelled';
 COMMENT ON COLUMN public.circulation_requests.fulfilment_preference IS 'How should the request be fulfilled (whether the item should be kept on the hold shelf for collection or delivered to the requester)';
 COMMENT ON COLUMN public.circulation_requests.hold_shelf_expiration_date IS 'Date when an item on the hold shelf expires';
 COMMENT ON COLUMN public.circulation_requests.holdings_record_id IS 'ID (generated UUID) of the holdings record being requested';
+COMMENT ON COLUMN public.circulation_requests.instance_title IS 'Title of the item being requested';
 COMMENT ON COLUMN public.circulation_requests.instance_id IS 'ID (generated UUID) of the instance being requested';
+COMMENT ON COLUMN public.circulation_requests.item_barcode IS 'Barcode of the item being requested';
 COMMENT ON COLUMN public.circulation_requests.item_id IS 'ID (generated UUID) of the item being requested';
+COMMENT ON COLUMN public.circulation_metadata_created_by_user_id IS 'ID (generated UUID) of the user who created the record (when available)';
+COMMENT ON COLUMN public.circulation_metadata_created_date IS 'Date and time when the record was created';
+COMMENT ON COLUMN public.circulation_metadata_updated_by_user_id IS 'ID (generated UUID) of the user who last updated the record (when available)';
+COMMENT ON COLUMN public.circulation_metadata_updated_date IS 'Date and time when the record was last updated';
```

Comment:
These appear to be table names rather than column names.

Reply:
Whoops! You're right. I goofed up. Looks like I just missed a word/decimal point. I'll take a look and make adjustments.